src/mon/OSDMonitor.cc

```diff
@@ -970,31 +970,192 @@ void OSDMonitor::encode_pending(MonitorDBStore::TransactionRef t)
     tmp.apply_incremental(pending_inc);
 
     if (tmp.require_osd_release >= CEPH_RELEASE_LUMINOUS) {
-      // set or clear full/nearfull?
+      // remove any legacy osdmap nearfull/full flags
+      {
```

Comment:
if (tmp.test_flag(CEPH_OSDMAP_FULL | CEPH_OSDMAP_NEARFULL))
so we don't see this msg every time?